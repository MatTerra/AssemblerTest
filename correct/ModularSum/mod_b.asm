MOD_B: BEGIN
section data
Y: SPACE
section text
OUTPUT Y
VAL: EXTERN
L1: EXTERN
PUBLIC Y
PUBLIC MOD_B
OUTPUT VAL
OUTPUT Y
JMP L1
END
