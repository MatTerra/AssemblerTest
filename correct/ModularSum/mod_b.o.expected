H: mod_b
H: 9
H: 010101010
H: U l1 7
H: U val 3
H: D mod_b 0
H: D y 8
T: 13 08 13 02 13 08 05 02 0
